src/main/java/net/dcess/gestrec/entity/Account.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package net.dcess.gestrec.entity;
+
+/**
+ *
+ * @author aaytouna
+ */
+public class Account {
+    
```

Comment:
account is empty ? how did you test ?

Reply:
i m waiting for the account class to be created